schema/device.json

```diff
@@ -1,6 +1,6 @@
 {
     "$schema": "https://json-schema.org/draft-07/schema",
-    "$id": "https://harp-tech.org/draft-02/schema/device.json",
+    "$id": "https://harp-tech.org/draft-03/schema/device.json",
```

Comment:
How about using the protocolVersion as the URL? we could easily fetch different versions this way. The potential problem are typos that need to be correct, but in that event I would just override the json anyway.

Reply:
Just to see if I get the idea correctly, you are suggesting we would use here something like:
```
"$id": "https://harp-tech.org/1.13.0/schema/device.json",
```

I fear this may be abusing the `$id` field a bit too much, since for this to work we would need to constantly update our "schema" version every time we add a new register (note that adding a new core register does not currently requiring changing the core JSON schema, only the core register metadata file, i.e. the `core.yml` and not the `core.json`).

I do see your point about where to store the `core.yml` long-term, and maybe it should indeed be stored in a different place than the JSON schemas themselves. I feel here we should try hard to avoid changing the core registers too frequently as this will very quickly become unwieldy and increase the chances for complications and backwards incompatibility.

I would try to gather all changes to the core registers we have now and make one big update which hopefully should last us for at least an entire year.

Reply:
Woops, you are right my mistaken. I would leave them together for now.